src/users/users.ts

```diff
@@ -392,12 +389,19 @@ export async function editUser(ctx: IContext, params: IParameters): Promise<IRes
 }
 
 export async function updateUser(ctx: IContext, params: IParameters, body: object): Promise<IResponse> {
-  await requirePermissions(ctx, 'org_manager', params);
   const cf = new CloudFoundryClient({
     accessToken: ctx.token.accessToken,
     apiEndpoint: ctx.app.cloudFoundryAPI,
   });
 
+  const isAdmin = ctx.token.hasScope(CLOUD_CONTROLLER_ADMIN);
+  const isManager = await cf.hasOrganizationRole(params.organizationGUID, ctx.token.userID, 'org_manager');
+
+  /* istanbul ignore next */
+  if (!isAdmin && !isManager) {
```

Comment:
Some of the changes in this commit don't seem related to signingKeys refactoring. Could you update the commit message to describe what it's doing? It feels a bit odd to partially refactor the signingKeys stuff here prior to the other story being worked.

Reply:
You've managed to make me doubt at what we're doing. I've reworded that commit entirely as I got confused at what we're trying to accomplish. See if that's better?